src/pages/StreamerPageMain.tsx

```diff
@@ -11,17 +11,25 @@ const StreamerPageMain = ({ player, videosById, videosSorted, clipsById, clipsSo
         <>
             {player &&
                 <div className="streamer_page">
+                    <h1>
                     <div className="streamer_page__avatar">
                         <img src={`http://streamsnipers.com/static/images/streamers/${player.name}.png`} alt={`${player.name} avatar`} />
-                        <h1>{player.name}</h1>
+                        {player.name}
                     </div>
+                    </h1>
                     {videosSorted.length > 0 &&
                         <>
-                            <h3>Recent one category</h3> <Link to={`/player/${player.id}/${player.slug}/videos`}>View All ></Link>
+                            <h3>
+                                Recent broadcasts
+                                <a className="small_link" href={`/player/${player.id}/${player.slug}/videos`}>view all</a>
```

Comment:
you can add className to link as well. link is tied to the router even after compiling it becomes a i think it add additional stuff

Reply:
yeah it doesnt reload url